src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Extensions/DefaultTagHelperTargetExtension.cs

```diff
@@ -650,10 +646,13 @@ private string GetContent(HtmlContentIntermediateNode node)
     // Internal for testing
     internal static string GetDeterministicId(CodeRenderingContext context)
```

Comment:
Should this be moved somewhere else? It's not used for tag helpers only anymore.